components/02-molecules/pop-out-image/pop-out-image.twig

```diff
@@ -41,3 +41,4 @@
     } %}
   </div>
 </div>
+<div {{ bem('clearfix', [], pop_out_image__base_class) }}></div>
```

Comment:
Should this be inside the inner or at least the main `pop-out-image` wrapper so it's closer to the floated image? I guess this works but it seems odd that it's on the very outside.

Reply:
Removed when I added the `::after` clearfix